tests/profile/test_user_configurable_profiler_v3_batch_request.py

```diff
@@ -978,6 +980,12 @@ def test_profiler_all_expectation_types_pandas(
     assert results["success"]
 
 
+@pytest.mark.skipif(
+    is_version_greater_or_equal(pd.__version__, "2.0.0"),
```

Comment:
Should this check both pyspark and pandas version?

Reply:
I thought a bit about this, and I think the check should only be pandas 2.0.0. it's because there currently is no `pyspark` version that is compatible with pandas 2.0, whereas *any* version of pandas less than 2.0.0 is compatible with `pyspark`.